code/game/machinery/hologram.dm

```diff
@@ -35,6 +35,7 @@ var/const/HOLOPAD_MODE = RANGE_BASED
 	name = "\improper AI holopad"
 	desc = "It's a floor-mounted device for projecting holographic images. It is activated remotely."
 	icon_state = "holopad0"
+	layer = 2.1

```

Comment:
2.1 is also the layer of crayon decal, which means drawing a crayon on the holopad's tile would show the drawing above the holopad, is it intended?